internal/utils/file/file_test.go

```diff
@@ -110,36 +111,47 @@ func TestCreateAndWriteFile(t *testing.T) {
 
 func TestGetDependencyCodeFilepathAndLine(t *testing.T) {
 	t.Run("Should run with success", func(t *testing.T) {
-		code, file, line, err := file.GetDependencyCodeFilepathAndLine(
-			testutil.CsharpExample1, "", "Microsoft.AspNetCore.Http", dotnetcli.CsProjExt,
+		dependencyInfo, err := file.GetDependencyCodeFilepathAndLine(
+			testutil.CsharpExample1, "", []string{"Microsoft.AspNetCore.Http", "2.2.2"}, dotnetcli.CsProjExt,
 		)
-
-		expectedCode := "<PackageReference Include=\"Microsoft.AspNetCore.Http\" Version=\"2.2.2\"/>"
+		expectedCode := "    <PackageReference Include=\"Microsoft.AspNetCore.Http\" Version=\"2.2.2\"/>"
 		expectedFile := filepath.Join(testutil.CsharpExample1, "NetCoreVulnerabilities", "NetCoreVulnerabilities.csproj")
 		expectedLine := "7"
 		assert.NoError(t, err)
-		assert.Equal(t, expectedLine, line)
-		assert.Equal(t, expectedFile, file)
-		assert.Equal(t, expectedCode, code)
+		assert.Equal(t, expectedLine, dependencyInfo.Line)
+		assert.Equal(t, expectedFile, dependencyInfo.Path)
+		assert.Equal(t, expectedCode, dependencyInfo.Code)
 	})
 	t.Run("Should return empty when path is invalid", func(t *testing.T) {
-		code, file, line, err := file.GetDependencyCodeFilepathAndLine(
-			"invalidPath", "", "Microsoft.AspNetCore.Http", dotnetcli.CsProjExt,
+		dependencyInfo, err := file.GetDependencyCodeFilepathAndLine(
+			"invalidPath", "", []string{"Microsoft.AspNetCore.Http"}, dotnetcli.CsProjExt,
 		)
-
 		assert.Error(t, err)
-		assert.Zero(t, code)
-		assert.Zero(t, file)
-		assert.Zero(t, line)
+		if runtime.GOOS == "windows" {
+			assert.Equal(t, "CreateFile invalidPath: The system cannot find the file specified.", err.Error())
+		} else {
+			assert.Equal(t, "lstat invalidPath: no such file or directory", err.Error())
+		}
```

Comment:
We could assert using `os.ErrNotExist` instead.

Reply:
but their values are not the same. Look the result of the test:
```
        	Error:      	Not equal: 
        	            	expected: "file does not exist"
        	            	actual  : "lstat invalidPath: no such file or directory"
```

Reply:
Are you using `assert.ErrorIs(t, err, os.ErrNotExist)`?

Reply:
yes

Reply:
Oh no! I'm using `Equals(t, err, os.ErrNotExist)`, maybe can be this! 
I'll change this :)